tests/test_crrej_from_raw.py

```diff
@@ -63,7 +63,7 @@ def teardown_class(cls):
 
     def setup_method(self, method):
         self.cwd = os.getcwd()
-        self.directory = TemporaryDirectory(prefix='test_crrej_from_raw_')
+        self.directory = TemporaryDirectory(prefix='test_crrej_from_raw_', ignore_cleanup_errors=True)
```

Comment:
IS it worth putting these TemporaryDirectory calls in context managers?  Or does the ignore_cleanup_errors=True mean that you don't get ResourceWarnings?  We've had problems in the jwst tests recently where unguarded opens and TemporaryDirectory calls have caused tests to fail with python 3.14.1 and later

Reply:
I was getting OSErrors prior to python 3.14.1, which was just released a week ago.

It seems to be related to cleaning up directories located on central store, and was mitigated by that flag:
```
OSError: [Errno 39] Directory not empty: '/.../scratch/test_crrej_from_raw_ib8ala8s'
```
But checking the directory after the tests are run shows no hidden or visible files.

Reply:
I don't think it's necessary to use the pytest context managers to generate the temp directories since the residual directories are empty.  I'm not sure this would eliminate the OSError anyway.